tbtadm/controller.cpp

```diff
@@ -94,9 +94,9 @@ enum security_level
 };
 
 const std::string symbols[] = {
-        [SYMBOL_PIPE] = "│",
-        [SYMBOL_L]    = "└─ ",
-        [SYMBOL_PLUS] = "├─ ",
+        "│",
+        "└─ ",
+        "├─ ",
```

Comment:
The point that Coverity doesn't understand GNU extensions is sad, but I think the solution should be to use an std::unordered_map, to get the same safety and clarity as the original code gave.

Reply:
(And if you are going to change this, it'd be nice to change the enum to `enum class`, again for better safety, and then `Symbol` will be the enum name and the entries don't need this prefix anymore.)

Reply:
@ybernat is this legal construction btw in C++? AFAIK this is not correct way. http://en.cppreference.com/w/cpp/language/aggregate_initialization

Reply:
> GNU extension

Reply:
Yes, this is also an option.